docs/reference/commandline/build.md

```diff
@@ -298,6 +298,9 @@ accessed like regular environment variables in the `RUN` instruction of the
 Dockerfile. Also, these values don't persist in the intermediate or final images
 like `ENV` values do.
 
+Using this flag will not alter the output you see when the `ENV` lines from the
```

Comment:
`ENV` or `ARG` ?  I thought you were worried that the `ARG` output would be confusing.